android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/SettingsFragment.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.physical_web.physicalweb;
+
+import android.os.Bundle;
+import android.preference.ListPreference;
+import android.preference.PreferenceFragment;
+import android.view.Menu;
+
+/**
+ * This fragment is the ui that the user sees when
+ * they have entered the app's settings mode.
+ * This ui is where the user can view the current configuration
+ * of the application (including it's default PWS client)
+ * and also allows the user to change the app's behavior.
+ */
+
+public class SettingsFragment extends PreferenceFragment{
+
+  private static final String TAG = "SettingsFragment";
+
+  public SettingsFragment() {
+  }
+
+  public static SettingsFragment newInstance() {
```

Comment:
I also thought it was strange. It seems to have no real value. I just did it this way to be consistent. I think what I should do is remove it and call the constructor explicitly, then make another pull request to make the other fragments consistent. Thoughts?